app/javascript/src/tooltips.js

```diff
@@ -0,0 +1,28 @@
+// Pertaining to tooltips
+$(document).on('turbolinks:load', () => {
+  const tooltipSpan = ' <span class="tooltip-header-help">(?)</span>';
+  // For regular spans
+  $('.daria-tooltip').tooltip();
+
+  // Some evil magic for things attached to labels with rails-bootstrap-form
+  $('.tooltip-header-input').each(function() {
+    $(this).append(tooltipSpan);
+    const text = $(this).parent().find('input')[0].dataset.tooltipText;
```

Comment:
The coffeescript didn't port cleanly on `tooltip-header-input` or `tooltip-header-checkbox` so I had to refactor a smidge.